### ISSUE TITLE
Global synchronization

### ISSUE THREAD

**Issue by Blankslide** [opensistemas-hub/osbrain#362]
Is there a way to achieve a global synchronization call as it is in MPI.Barrier() in mpi4py?

**Comment by Peque** [comment]
@Folorunblues That pattern, and many others, are not implemented inside osBrain.

Since osBrain is based on ZeroMQ you can, however, implement any communication pattern you need without requiring too much effort. I strongly recommend you to have a look at [ØMQ - The Guide](https://zguide.zeromq.org/). There you will find lots of examples for different communication patterns for different needs. :blush:

**Comment by Blankslide** [comment]
@Peque Thank you for your prompt reply! I will take a look at the guide. Is osbrain 0.70 coming out any time soon?

**Comment by Peque** [comment]
@Folorunblues I don't think so. Unfortunately, I am no longer actively developing this library, and it seems the project is not receiving lots of contributions either.

**Comment by Blankslide** [comment]
@Peque, I quite understand. I believe the 0.6.5 is great and still has all the necessary features. Thanks again for your response!

**Comment by Peque** [comment]
@Folorunblues Glad you find it useful! :blush: 

Let me keep the issue open for contributions.
